### ISSUE TITLE
Aspects dropped by ZetAspects aren't given credit by this mod

### ISSUE THREAD

**Issue by LordVGames** [william-hollywood/ThatWasMine#3]
When an aspect is dropped, the message from ZetAspects is there, but not this mod's message. This might also apply to normal aspects, but I haven't tested it. I have tested the ZetAspect drop both with and without the sacrifice artifact, in multiplayer with another person, and by myself with the debug option on, and the message never appeared for the ZetAspect drop.

**Comment by william-hollywood** [comment]
Yeah, those aspects have always not worked, but they're so uncommon that I never really bothered (without ZetAspects that is, IIRC ZetAspects makes them more common to drop). 

The logic/decision for how they are dropped is different to the sacrifice artifact drop.

I'll have a look into it and how it works for vanilla aspects as well as if it's easy enough to allow zet ones to work.

**Comment by william-hollywood** [comment]
Had a quick look. Would be kind of difficult (I'm currently unsure how) to get it working for ZetAspects. 

Doesn't seem like too much of a mission for the Vanilla side though. If i get a spare moment I'll give it a crack.

I may come up with something for ZetAspects too, who knows.
